.github/workflows/markdown_to_pdf.yml

```diff
@@ -0,0 +1,31 @@
+name: Docs to PDF
+on:
+  push:
+    branches:
+      - master
+    # Paths can be used to only trigger actions when you have edited certain files, such as a file within the /docs directory
+    paths:
+      - 'src/**.md'
+      - 'src/**'
+      
+  workflow_dispatch:
+    
+jobs:
+  converttopdf:
+    name: Build PDF
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: baileyjm02/markdown-to-pdf@v1
+        with:
+          input_dir: src
+          output_dir: pdfs
+          images_dir: src/
+          # for example <img src="./images/file-name.png">
+          image_import: ./
+          # Default is true, can set to false to only get PDF files
+          build_html: false
+      - uses: actions/upload-artifact@v1
+        with:
+          name: docs
```

Comment:
Is this going to be the final file name? Does this require an extension? (I'm kinda lost on this.)

If so, it would be nice to have a file name that really shows its origin, such as `overreact-guide.pdf` or something. People are going to download that and blindly send it as an attachment, etc. and they never know what's in the file 😆